app/assets/stylesheets/application.scss

```diff
@@ -33,6 +33,21 @@ footer {
   padding-top: 15px;
 }
 
+.page-nav {
+  margin-top: 35px;
+  padding-left: 15px;
+  background-color: gray;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.